templates/extra/resources.html

```diff
@@ -0,0 +1,289 @@
+{% extends "base.html" %}
+{% load staticfiles %}
+{% block extrastyle %}
+    <link rel="stylesheet" href="{% static "css/extra/resources.css" %}" type="text/css" charset="utf-8" />
+{% endblock %}
+{% block content %}
+    <h2 class="extra-padding">Artists Resources</h2>
+    <div class="break">
+        <hr>
+    </div>
+    <div class="row">
+        <div class="columns small-12">
+            <div class="columns small-2 float-left" data-sticky-container>
+                <div class="sticky" data-sticky data-anchor="example1">
+                    <Ul>
+                        <!-- Overview -->
+                        <ul class="vertical menu content-box" data-accordion-menu>
+                            <li>
+                                <a class="content-header" href="overview-nested">Overview</a>
+                                <ul class="menu vertical nested">
+                                    <li><p class="content-sub"><a href=#overview-1>What is PerDiem</a></p></li>
```

Comment:
All of your links should be surrounded by quotes.
```html
<a href="#overview-1">
```
not
```html
<a href=#overview-1>
```

As is, these links probably won't work correctly on some browsers.